src/SFA.DAS.AssessorService.ExternalApiDataSync/Infrastructure/ConfigurationHelper.cs

```diff
@@ -39,5 +39,10 @@ public static IWebConfiguration GetConfiguration()
 
             return _webConfiguration;
         }
+
+        public static string GetEnvironmentName()
```

Comment:
See comment about about this

Reply:
It's to do with it being a Function app and I couldn't find an injected IHostingEnvironment